packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -48,11 +48,11 @@ typedef _ContextMenuPreviewBuilderChildless = Widget Function(
 Rect _getRect(GlobalKey globalKey) {
   assert(globalKey.currentContext != null);
   final RenderBox renderBoxContainer = globalKey.currentContext!.findRenderObject()! as RenderBox;
-  return Rect.fromPoints(renderBoxContainer.localToGlobal(
-    renderBoxContainer.paintBounds.topLeft,
-  ), renderBoxContainer.localToGlobal(
-    renderBoxContainer.paintBounds.bottomRight
-  ));
+  return Rect.fromPoints(
```

Comment:
🤔  not sure what happened here

Reply:
Oh I see, this is a case where a trailing comma would help:

```
  return Rect.fromPoints(
      renderBoxContainer.localToGlobal(
        renderBoxContainer.paintBounds.topLeft,
      ),
      renderBoxContainer.localToGlobal(renderBoxContainer.paintBounds.bottomRight));
 ```

Reply:
and 2 trailing commas would help more :)

```dart
  return Rect.fromPoints(
    renderBoxContainer.localToGlobal(
      renderBoxContainer.paintBounds.topLeft,
    ),
    renderBoxContainer.localToGlobal(
      renderBoxContainer.paintBounds.bottomRight,
    ),
  );
 ```

or 

```dart
  return Rect.fromPoints(
    renderBoxContainer.localToGlobal(renderBoxContainer.paintBounds.topLeft),
    renderBoxContainer.localToGlobal(renderBoxContainer.paintBounds.bottomRight),
  );
 ```